web/src/microapps.ts

```diff
@@ -36,8 +36,9 @@ export const decodeToken = (idToken: string) =>
  * Gets microapps identity token and its decoded form.
  * Returns the cached identity if it exists and token has not expired,
  * else, fetches a new identity token from the microapps API.
+ * Returns undefined if calling the getIdentity API fails.
```

Comment:
when will this happen? user rejects permission?

Reply:
@ifhuang Yup that's one of the cases that results the getIdentity API to throw an error